pkg/ddevapp/wordpress.go

```diff
@@ -260,3 +326,40 @@ func wordpressImportFilesAction(app *DdevApp, importPath, extPath string) error
 
 	return nil
 }
+
+// getRelativeAbsPath returns the portion of the ABSPATH value that will come after "/" in wp-config.php -
+// this is done by searching (at a max depth of one directory from the docroot) for wp-settings.php, the
+// file we're using as a signal to indicate that this is a WordPress project.
+func getRelativeAbsPath(app *DdevApp) (string, error) {
```

Comment:
I think you'd be much happier here with a very simple `matches, err := filepath.Glob("*/<string>")` - I haven't tried that globbing directories, but I'd think it would work.

If this is just about Wordpress, let's name it as being for wordpress. It looks like it's intended to be a general-purpose func by the naming. But I do think you'll like the glob.

Reply:
Good point on renaming, I'll make that change.

I'm looking into updating this to use filepath.Glob() now -- it's not _quite_ as simple as that example as that _only_ looks in subdirectories, but I think there's some simplification there.

Reply:
Hope you saw the example on github, which looks exactly one directory down for a file.